sbol_utilities/default_components.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Tuple
+
+import sbol3
+import tyto
+
+def promoter(name: str, description: str, sequence: str)-> Tuple[sbol3.Component, sbol3.Sequence]:
+    """
+    Creates a promoter and a sequence Components with a given name, description, and sequence.
```

Comment:
In this an all of the other routines, the docstring should include input and output argument information as well. See the examples in `sbol_utilities.component`

Reply:
I will improve the docstrings following the `sbol_utilities.component`

Reply:
> Somewhere, maybe in an email thread, you mentioned adding things to a document as needing to be separate calls. This has changed recently in pySBOL3. `Document.add` takes a single top level or a sequence of TopLevel objects. So you can do a single add to get all of the produced items into the document.
> 
> I don't see an `add` in the pull request, so maybe that would go in example code or documentation or something like that.

I saw the implementation on the pySBOL3 repo to add all, but apparently it was not implemented when I imported the package. I clone and use the pySBOL repo and that have the feature. Maybe it is a version problem.